helpdesk_mgmtsystem_nonconformity/models/mgmtsystem_nonconformity.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class MgmtsystemNonconformity(models.Model):
+    _inherit = "mgmtsystem.nonconformity"
+
+    ticket_id = fields.Many2one(comodel_name="helpdesk.ticket", string="Ticket",)
```

Comment:
This should be a one2many, the inverse of the other m2o, as you can link several helpdesk tickets (several notifications of the same problem), to the same non conformity.